docs/ha-mode.md

```diff
@@ -5,10 +5,6 @@ The following components require a highly available endpoints:
 * etcd cluster,
 * kube-apiserver service instances.
 
-The former provides the
-[etcd-proxy](https://coreos.com/etcd/docs/latest/proxy.html) service to access
-the cluster members in HA fashion.
-
```

Comment:
Please note instead that the former doesn't benefit from reverse proxies as all cluster components may now access multiple etcd endpoints, which provides HA as well.